server/container_create.go

```diff
@@ -386,8 +386,7 @@ func ensureSaneLogPath(logPath string) error {
 
 // addSecretsBindMounts mounts user defined secrets to the container
 func addSecretsBindMounts(mountLabel, ctrRunDir, configDefaultMountsPath string, specgen generate.Generator) error {
```

Comment:
Should we check explicitly in the caller for empty string as setting and skip calling this altogether?

Reply:
fixed